addon/components/power-select/options.ts

```diff
@@ -32,8 +32,8 @@ if(typeof FastBoot === 'undefined'){
 }
 
 export default class Options extends Component<Args> {
-  private isTouchDevice = isTouchDevice
-  private hasMoved = false
+  private isTouchDevice = (this.args as any)?.extra?._isTouchDevice || isTouchDevice
```

Comment:
do you need this? In the `testem.js` file we seem to start the browser with the `--touch-events` flag, so that should be enough, right?

Reply:
Not sure what other approach can be taken here. In [ember-basic-dropdown](https://github.com/cibernox/ember-basic-dropdown/pull/582/files#diff-47a26d7d1178b224b35da72ba32c96aea5490b6bfa7662fbd52286e1ed39a083R286) this option is a part of component API... Not sure if that's better honestly 😅 

But yeah, this is only needed for testing.